### PR TITLE
Remove PHP 5 from mysqli, PDO_MySQL and mysqlnd

### PR DESCRIPTION
**Pull request by kamil-tekiela**
I was not sure what to do about https://www.php.net/manual/en/mysqli.installation.php

### CODE REVIEW

**Review by afilina — CHANGES_REQUESTED** [review]
Besides one nitpicky suggestions, LGTM.

File: reference/mysqli/constants.xml [hunk 1]
```diff
@@ -68,15 +66,14 @@
...
-      Only valid for mysqlnd. Available since PHP 5.3.0.
+      Only valid for mysqlnd.
      </para>
     </listitem>
    </varlistentry>
    <varlistentry xml:id="constantmysqli-opt-ssl-verify-server-cert">
     <term><constant>MYSQLI_OPT_SSL_VERIFY_SERVER_CERT</constant></term>
     <listitem>
      <para>
-      Available since PHP 5.3.0. (MySQL 5.1.10 and up)
```
Comment (cmb69) [inline reference/mysqli/constants.xml:79]: Sadly, PHP 7.0.0 is supposed to [work with MySQL >= 4.1.13 and MySQL >= 5.0.7](https://github.com/php/php-src/blob/php-7.0.0/ext/mysqli/config.m4#L87), so please keep the relevant info for MySQL.

File: reference/mysqli/book.xml [hunk 2]
```diff
@@ -60,12 +60,6 @@
  &reference.mysqli.mysqli-sql-exception;
  &reference.mysqli.reference;
 
-<appendix xmlns="http://docbook.org/ns/docbook" xml:id="changelog.mysqli">
- <title>&ChangelogListingTitle;</title>
- <para>&ChangelogListingDescription;</para>
- <?phpdoc generate-changelog-for="ref.mysqli class.mysqli class.mysqli-stmt class.mysqli-result class.mysqli-driver class.mysqli-warning class.mysqli-sql-exception"?>
-</appendix>
```
Comment (cmb69) [inline reference/mysqli/book.xml:67]: Even if there are no respective changelogs *currently*, I think we should leave that appendix (may comment it out).

**Review by cmb69 — COMMENTED** [review]
Thanks! Overall a very nice cleanup. Please check my review comments.

File: reference/mysqli/book.xml (hunk 2 above)
Comment (kamil-tekiela) [reply reference/mysqli/book.xml:67]: Are you sure about this? I was really surprised when I discovered [this section](https://www.php.net/manual/en/changelog.mysqli.php). I believe that all changes should be logged under the functions that they belong too. Not many people will go looking through the manual to find some hidden page that contains a list of all the changes. Besides, it looks like nobody was maintaining it anyway. If we leave it there, how will we make sure that people update them when they update the manual? I haven't found such section in other books in the manual.

File: reference/mysqli/constants.xml (hunk 1 above)
Comment (kamil-tekiela) [reply reference/mysqli/constants.xml:79]: I'll review it over the weekend and add it back in. Thanks!

File: reference/mysqli/book.xml (hunk 2 above)
Comment (cmb69) [reply reference/mysqli/book.xml:67]: That page is autogenerated by the `<?php generate-changelog-for?>` processing instruction. IMO a nice feature, that can be helpful to get an overview.

File: reference/mysqli/constants.xml (hunk 1 above)
Comment (kamil-tekiela) [reply reference/mysqli/constants.xml:79]: Added the note + 2 other back in.

File: reference/mysqli/book.xml (hunk 2 above)
Comment (kamil-tekiela) [reply reference/mysqli/book.xml]: Reverted the commit

**Comment by afilina** [comment]
Checked the new commits. Looks like all suggestions were addressed.

**Comment by cmb69** [comment]
Thank you!

**Pull request closed without merging**